apps/fm/js/fm.js

```diff
@@ -619,12 +622,16 @@ var favoritesList = {
     elem.id = this._getUIElemId(item);
     elem.className = 'fav-list-item';
     elem.setAttribute('role', 'option');
-    var html = '';
-    html += '<div class="fav-list-frequency">';
-    html += item.frequency.toFixed(1);
-    html += '</div>';
-    html += '<div class="fav-list-remove-button"></div>';
-    elem.innerHTML = html;
+
+    var subElem = document.createElement('div');
+    subElem.className = 'fav-list-frequency';
+    subElem.textContent = item.frequency.toFixed(1);
+    subElem.dataset.unit = mhzString;
```

Comment:
When translation is missing we will display string ID (`mhz` in this case), so the only scenario in which it will be displayed is when DOMLocalized is never fired.

One more thing to consider is that means that we always switch from "Mhz-NOTLOCALIZED" to localized Mhz string which may trigger reflow, but I guess that it's not a significant performance cost here at all.

Anyway, I'm not strongly opinionated here :)